stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelController.kt

```diff
@@ -26,7 +27,7 @@ public interface ChannelController {
     public val channelType: String
     public val channelId: String
     /** a list of messages sorted by message.createdAt */
-    public val messages: LiveData<List<Message>>
```

Comment:
It is a breaking change in our public API.
Apart of that, it is breaking the previous comment and the KDocs over the interface.
Why don't we provide it into another property?

Reply:
If a property must be deprecated, a proper `deprecated` annotation should be added, explaining what and how to use instead.

Reply:
I don't mind providing it in another property, but I don't see another way. I need to differentiate new messages from old ones so the property `loadingMore`  from `MessageListItemWrapper` gets the right information... until now it always get false.

Reply:
This "public livedata" could be a transformation of the new livedata you are creating.
We keep compatibility and the new feature is done

Reply:
Without changing `public val messages: LiveData<List<Message>>` I don't see a way to differentiate new messages from old messages that were loaded from the history of the conversation.